components/switch/switch.js

```diff
@@ -0,0 +1,155 @@
+import '../colors/colors.js';
+import '../icons/icon.js';
+import '../icons/icon-custom.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+
+class Switch extends LitElement {
+
+	static get properties() {
+		return {
+			label: { type: String },
+			labelHidden: { type: Boolean, attribute: 'label-hidden' },
+			offIcon: { type: String, attribute: 'off-icon' },
+			on: { type: Boolean, reflect: true },
+			onIcon: { type: String, attribute: 'on-icon' },
+			role: { type: String, reflect: true },
+			tabindex: { type: String, reflect: true }
```

Comment:
Should we go with `reflect: true` for everything seeing as it's our recommendation in the best practices?